src/cli.ts

```diff
@@ -232,7 +243,6 @@ const main = async (options: NormalizedOptions) => {
 
   // execute "Step 1": bulk data export
   if (options.fhirUrl && options.group) {
-    await executeExport();
```

Comment:
Suggest keeping this line since it calls the kickoff for bulk export - otherwise the export and download processes will never occur

Reply:
Ouch. I found I removed the line for mistake. I will put it back. Sorry about confusion.